packages/react-components/src/components/VideoTile.tsx

```diff
@@ -404,6 +427,41 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
           </Stack>
         )}
 
+        {
+          /* @conditional-compile-remove(reaction) */
+          canRenderReaction && (
+            <Stack
+              className={mergeStyles(videoContainerStyles, {
+                display: 'flex',
```

Comment:
nit: we try to keep all styles in a `<name>.styles.ts` files to keep these files as short as possible